production/scrapers/texas_deaths_tji.R

```diff
@@ -0,0 +1,111 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+texas_deaths_tji_pull <- function(x){
+    "1mOS1wggvyRUOpI-u2VabmnQ1yJPPEgOc2zdZjWxbAwQ" %>% 
+        googlesheets4::read_sheet(
+            sheet = "Inmate Deaths", 
+            col_types = "cDDccccccccccccccc")
+}
+
+texas_deaths_tji_restruct <- function(x){
+    x_ <- x %>% 
+        filter(str_detect(FacilityType, "(?i)state|county")) 
+    
+    fac_types <- x_ %>% 
+        distinct(FacilityType) %>% 
+        pull()
+
+    for (i in fac_types){
+        if (!i %in% c("State Prison", "State Jail", "County Jail")){
+            stop(paste("Unexpected facility type:", i))
+        }
+    }
+    
+    if (length(fac_types) != 3){
+        stop(paste("Should have pulled State Prison and State Jail facilities.", 
+                    "Instead pulled:", toString(fac_types)))
+    }
+
+    x_ %>% 
+        mutate(Facility = paste(Facility, FacilityType)) %>% 
```

Comment:
Doing this so `Gist Unit State Jail` and `Gist Unit State Prison` can get assigned to the correct jurisdiction downstream!